package.json

```diff
@@ -59,6 +59,7 @@
     "jest-image-snapshot": "^2.7.0",
     "jest-puppeteer": "^4.3.0",
     "puppeteer": "^2.0.0",
+    "resize-observer": "^1.0.0",
```

Comment:
If anything, it should be a normal dependency since you import it in the lib's code but I think we should not use it at all (my other comment).

Reply:
The problem with not having the the `resize-observer` dependency is, that TypeScript does not know about ResizeObserver yet: https://github.com/microsoft/TypeScript/issues/37861
I am not familiar with TypeScript - maybe you know a better solution?

Reply:
We can just cast it to `any`. It's not an exposed through the lib's API anyway.